app/js/controllers/foyer/pensionsAlimentaires.js

```diff
@@ -51,34 +50,34 @@ angular.module('ddsApp').controller('FoyerPensionsAlimentairesCtrl', function($s
         $scope.individusVM.push(createIndividuVM(conjoint));
     }
 
-    $scope.hasPensionsAlimentaires = false;
+    $scope.situation.parentsPayPensionsAlimentaires = false;
     $scope.individusVM.forEach(function(individuVM) {
         if (_.any([
             _.find(individuVM.individu.ressources, { type: 'pensionsAlimentairesVersees' })
         ])) {
-            $scope.hasPensionsAlimentaires = true;
+            $scope.situation.parentsPayPensionsAlimentaires = true;
         }
     });
 
-    var applyPensionToIndividu = function(individuVM, typePension) {
+    function applyPensionToIndividu (individuVM) {
         var individu = individuVM.individu;
-        var ressource = individuVM[typePension];
+        var ressource = individuVM.pensionsVersees;
         RessourceService.spreadIndividuRessources(individu, months, ressource, $scope.situation.dateDeValeur);
-    };
+    }
 
-    $scope.submit = function() {
-        $scope.individusVM.forEach(function(individuVM) {
-            if ($scope.hasPensionsAlimentaires) {
-                ['pensionsVersees'].forEach(function(typePension) {
-                    applyPensionToIndividu(individuVM, typePension);
-                });
-            } else {
+    $scope.submit = function(form) {
+        form.submitted = true;
```

Comment:
[This](https://github.com/sgmap/mes-aides-ui/blob/master/app/views/partials/foyer/capture-montant-ressource.html#L51) is included in the view ;)